### PR TITLE
[ZH] Add `Beatmap/Packs`

### PR DESCRIPTION
**Pull request by CloneWith**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)
- [x] Pull #9678 merged

### CODE REVIEW

**Comment by Sitoria** [comment]
PR approved

File: wiki/Beatmap/Packs/zh.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+tags:
+  - beatmap packs
+  - pack
+  - 曲包
```
Comment (YumeMuzi) [inline wiki/Beatmap/Packs/zh.md:5]:
```suggestion
  - 曲包
  - 谱面包
```

File: wiki/Beatmap/Packs/zh.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+| 标识 | 类别 | 谱面类型 |
+| :-: | :-- | :-- |
+| **S** | 常规 | [上架 (Ranked)](/wiki/Beatmap/Category#上架-(ranked)) 或[达标 (Approved)](/wiki/Beatmap/Category#达标-(approved)) 的谱面 |
+| **F** | 精选艺术家 | 来自[精选艺术家](/wiki/People/Featured_Artists)的歌曲 |
+| **T** | 锦标赛 | 锦标赛图池谱面 |
+| **L** | 社区喜爱计划 | 社区在[社区喜爱计划](/wiki/Community/Project_Loved)选出的谱面 |
+| **R** | 聚光灯 | 来自[聚光灯](/wiki/Beatmap_Spotlights)的季度挑战谱面曲包，过去被称为 *Charts* |
+| **T** | 主题 | 基于特别主题的谱面 |
+| **A** | 艺术家/专辑 | 来自特定艺术家或专辑的歌曲 |
```
Comment (YumeMuzi) [inline wiki/Beatmap/Packs/zh.md:38]:
```suggestion
| **A** | 艺术家/专辑 | 谱面内的歌曲来自特定艺术家或他们的专辑 |
```

File: wiki/Beatmap/Packs/zh.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+- [(Un)official Beatmap Pack Sheet](https://osu.ppy.sh/community/forums/topics/1528191)
+  - [Shige-Tori\[a\]](https://osu.ppy.sh/users/4459449) 的 [Google 表格](https://docs.google.com/spreadsheets/d/1gcXL9gubcWEKY1X2taxJdBGjFrqEpkNmjHU7LFpcJRo)
+- 使用 `packs.ppy.sh` 域名前的早期存储站点 ( ~2023.05.07)
+  - [Stefan](https://osu.ppy.sh/users/626907) 的 [MEGA 网盘](https://mega.nz/folder/Rl4hkKZQ#L1LXE4UgMH00eJF1xqMytw)
+- [osu! Beatmap Pack Mirror v2](https://osu.ppy.sh/community/forums/topics/57381) （不活跃）
+  - [nanashirei](https://osu.ppy.sh/users/807630) 的[镜像网站](http://osu.yas-online.net/)
+
+### 用 Torrent 下载
+
+如果想要批量下载曲包而非一个个下载它们，你可以使用 Torrent 工具，从以下站点下载：
```
Comment (YumeMuzi) [inline wiki/Beatmap/Packs/zh.md:65]:
```suggestion
### 用种子下载

如果你想要批量下载曲包，而非一个个下载它们，可以使用种子工具，并从以下站点下载：
```

File: wiki/Beatmap/Packs/zh.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+- [Osu! Beatmap Packs @ Resilio Sync](https://osu.ppy.sh/community/forums/topics/1255023)
+  - [glukki](https://osu.ppy.sh/users/14285150) 的 [Resilio Sync 上的 Torrent 文件](https://link.resilio.com/#f=osu%21%20Beatmap%20Packs&sz=19E2&t=1&s=JHR4G3EUWCAOAKJT6HITFDGMENTSXU7U&i=CASDYUCU4VP4JUMPRYFZLFZK5EIXANSEE&v=2.7&a=2)
```
Comment (YumeMuzi) [inline wiki/Beatmap/Packs/zh.md:68]: Torrent 改成种子

**Review by YumeMuzi — CHANGES_REQUESTED** [review]

**Review by YumeMuzi — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**